docs/src/resources/two-side-column.md

```diff
@@ -0,0 +1,841 @@
+}
+println(getVerilogString(new PassthroughGenerator(10)))
+println(getVerilogString(new PassthroughGenerator(20)))
+```
+</td>
+<td>
+
+```
+module PassthroughGenerator(
+  input        clock,
+  input        reset,
+  input  [9:0] io_in,
+  output [9:0] io_out
+);
+  assign io_out = io_in; // @[main.scala 13:10]
+endmodule
+
+module PassthroughGenerator(
+input         clock,
+input         reset,
+input  [19:0] io_in,
+output [19:0] io_out
+);
+assign io_out = io_in; // @[main.scala 13:10]
+endmodule
+```
+</td>
+         </tr>
+    </table>
+</body>
+</html>
+
+# Parameterizing a Module
+
+<html>
+<body>
+    <table border ="0">
+          <tr>
+            <td><b style="font-size:30px">Verilog</b></td>
+            <td><b style="font-size:30px">Chisel</b></td>
+            <td><b style="font-size:30px">Generated Verilog</b></td>
+         </tr>
+         <tr>
+<td>
+
+```
+module ParameterizedWidthAdder(
+
+input [in0Width-1:0] in0,
+input [in1Width-1:0] in1,
+output [sumWidth-1:0] sum
+);
+parameter in0Width = 8;
+parameter in1Width = 1;
+parameter sumWidth = 9;
+
+assign sum = in0 + in1;
+
+endmodule
+```
+
+</td>
+<td>
+
+```
+class ParameterizedWidthAdder(in0Width: Int, in1Width: Int, sumWidth: Int) extends Module {
+  require(in0Width >= 0)
+  require(in1Width >= 0)
+  require(sumWidth >= 0)
+  val io = IO(new Bundle {
+    val in0 = Input(UInt(in0Width.W))
+    val in1 = Input(UInt(in1Width.W))
+    val sum = Output(UInt(sumWidth.W))
+  })
+  // a +& b includes the carry, a + b does not
+  io.sum := io.in0 +& io.in1
+}
+```
+</td>
+             <td>Text comes here</td>
+</tr>
+<tr>
+<td>
+
+```
+module TestBench;
+wire [31:0] sum;
+ParameterizedWidthAdder  #(32, 32, 32) my32BitAdderWithTruncation (32'b0, 32'b0, sum);
+endmodule
+```
+</td>
+            <td>
+
+```
+val my32BitAdderWithTruncation = Module(new ParameterizedWidthAdder(32, 32, 32)
+```
+</td>
+             <td>Text comes here</td>
+         </tr>
+ <tr>
+<td>
+Text comes here
+</td>
+<td>
+
+```scala mdoc:silent
+class MyModule extends Module {
+  val io = IO(new Bundle {
+    val in  = Input(UInt(4.W))
+    val out = Output(UInt(4.W))
+  })
+
+val two  = 1 + 1
+println(two)
+val utwo = 1.U + 1.U
+println(utwo)
+
+io.out := io.in
+}
+println(getVerilogString(new MyModule))
+```
+</td>
+<td>
+
+```
+2
+UInt<1>(OpResult in MyModule)
+module MyModule(
+  input        clock,
+  input        reset,
+  input  [3:0] io_in,
+  output [3:0] io_out
+);
+  assign io_out = io_in; // @[main.scala 19:10]
+endmodule
+```
+</td>
+         </tr>
+    </table>
+<html>
+<body>
+
+# Wire assignment
+
+<html>
+<body>
+    <table border ="0">
+          <tr>
+            <td><b style="font-size:30px">Verilog</b></td>
+            <td><b style="font-size:30px">Chisel</b></td>
+            <td><b style="font-size:30px">Generated Verilog</b></td>
+         </tr>
+         <tr>
+<td>
+
+```
+wire [31:0] 
+a = 32'd42; 
+wire [31:0] 
+b = 32'hbabecafe; 
+wire [15:0] c; 
+assign c = 16'b1;
```

Comment:
For this section, can we include all the examples from the Cheatsheet showing the bit selects as well:

<img width="338" alt="Screen Shot 2022-01-13 at 9 17 58 AM" src="https://user-images.githubusercontent.com/17858596/149377819-21e8fc3e-0a62-42cc-8fa0-bc8cbad973d2.png">

Link to better cheat sheet source: https://www.overleaf.com/project/5cf6b7bc17e6773ee4149f7e

Reply:
https://scastie.scala-lang.org/Shorla/3tMpKXSiQ762Rqt2k0Ag8A/13
The adjustments I made. yet, to make it connect effectively.

Reply:
updated scastie: https://scastie.scala-lang.org/Shorla/3tMpKXSiQ762Rqt2k0Ag8A/27